src/a11y-dialog.ts

```diff
@@ -98,7 +98,7 @@ export default class A11yDialog {
     // stays trapped inside the dialog while open, and start listening for some
     // specific key presses (TAB and ESC)
     document.body.addEventListener('focus', this.maintainFocus, true)
-    document.addEventListener('keydown', this.bindKeypress)
```

Comment:
🔍 Listening for this event during the capturing phase is just a little 🤏🏻 faster, since the capturing phase runs first

Reply:
Heh that’s fair enough. I guess we really only need to listen to TAB keys in the container itself. It shouldn’t break anything.